Source/Core/Gen5/Generators/IDGenerator5.cpp

```diff
@@ -33,12 +33,11 @@ std::vector<IDState5> IDGenerator5::generate(u64 seed, u32 pid, bool checkPID)
 
     bool pidBit = (pid >> 31) ^ (pid & 1);
     u16 psv = (pid >> 16) ^ (pid & 0xffff);
-    u16 xorPSV = psv ^ 0x8000;
```

Comment:
Why was this removed

Reply:
Cause we don't need to check IDs combo for the xored PID. We're trying to get IDs combo to make the inserted PID shiny, not the xored PID